src/ui/ons/partials/ONSRadio.svelte

```diff
@@ -1,15 +1,25 @@
 <script>
   import { getContext } from "svelte";
 
-  export let id;
-  export let value;
-  export let checked = false;
+  export let id, value, radioValue, onChange;
   let name = getContext("name");
+  let userSelection;
```

Comment:
See if you can strip out this variable and just use `radioValue` - often it can make things clearer assigning to new variables but the extra step confused me here.